lib/changes_support/disjoint_list.rb

```diff
@@ -1,5 +1,3 @@
-require 'google_hash'
-
 module ChangesSupport
 end
 
```

Comment:
I'd have loved to added some documentation here, but I'm still a little confused about what this is. Definitely something to revisit.